Content.Server/AlertLevel/AlertLevelOnPressComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Shared.AlertLevelOnPress;
+using Robust.Shared.Audio;
+
+namespace Content.Server.AlertLevelOnPress;
+
+[RegisterComponent]
+[Access(typeof(AlertLevelOnPressSystem))]
+public sealed partial class AlertLevelOnPressComponent : SharedAlertLevelOnPressComponent
+{
+    [DataField(required: true)]
+    public string AlertLevelOnActivate = default!;
+
+    [DataField(required: true)]
+    public string TextField = default!;
```

Comment:
This is never used?

Reply:
fairlysadpanda suggested it above... but i dont really know what to do with it either, so ill remove it